hal/common/retarget.cpp

```diff
@@ -717,3 +719,34 @@ extern "C" void __env_unlock( struct _reent *_r )
 #endif
 
 } // namespace mbed
+
+void* operator new  ( std::size_t count )
+{
+    void *buffer = malloc(count);
+    if (NULL == buffer) {
+        error("Operator new out of memory\r\n");
+    }
+    return buffer;
+}
+
+void* operator new[]( std::size_t count )
```

Comment:
``( std::size_t count )`` -> ``(std::size_t count)`` in this commit all the changes shall be aligned